fixtures/src/components/controlled/checkbox-group.js

```diff
@@ -0,0 +1,93 @@
+        return (
+          <React.Fragment>
+            <h1>Controlled Checkbox Group</h1>
+            <h2>Unchecked</h2>
+            <div id='controlled_checkbox_group'>
+              <ul>
+                <li>onChange handler should update the log when each checkbox is checked</li>
+                <li>onBlur handler should update the log when each checkbox is blurred</li>
+                <li>each checkbox button has props value, name, id which should be reflected in both events</li>
+              </ul>
+              <CheckboxGroup
+                name='controlled_checkbox_group' label='Controlled Example'
+                onChange={ onChange }
+                onBlur={ onBlur }
+              >
+                <Checkbox
+                  id='controlled_checkbox_group_one'
+                  value='value one'
+                  label='label one'
+                  name='controlled_checkbox_group_one'
+                  checked={ checked.controlled_checkbox_group_one }
+                />
+                <Checkbox
+                  id='controlled_checkbox_group_two'
+                  value='value two'
+                  label='label two'
+                  checked={ checked.controlled_checkbox_group_two }
+                />
+                <Checkbox
+                  id='controlled_checkbox_group_three'
+                  value='value three'
+                  label='label three'
+                  checked={ checked.controlled_checkbox_group_three }
+                />
+              </CheckboxGroup>
+            </div>
+            <div id='controlled_checkbox_group_default'>
+              <h2>Default Checked</h2>
+              <ul>
+                <li>value two is checked by default</li>
+              </ul>
+              <CheckboxGroup
+                name='controlled_checkbox_group_default' label='Controlled Example default'
+                onChange={ onChange }
+                onBlur={ onBlur }
+              >
+                <Checkbox
+                  id='controlled_checkbox_group_default_one'
+                  value='value one'
+                  label='label one'
+                  checked={ checked.controlled_checkbox_group_default_one }
+                />
+                <Checkbox
+                  id='controlled_checkbox_group_default_two'
+                  value='value two'
+                  label='label two'
+                  checked={ checked.controlled_checkbox_group_default_two }
+                />
+                <Checkbox
+                  id='controlled_checkbox_group_default_three'
+                  value='value three'
+                  label='label three'
+                  checked={ checked.controlled_checkbox_group_default_three }
+                />
+              </CheckboxGroup>
+            </div>
+          </React.Fragment>
+        );
+      }}
+    </LogConsumer>
+  );
+};
+
+export default React.memo(ControlledCheckbox, () => false);
```

Comment:
Since this [`areEqual()` custom comparison](https://reactjs.org/docs/react-api.html#reactmemo) arrow function always returns `false`, doesn't that mean that `React.memo()` will never memoise the rendered result? If so, what's the purpose of using `React.memo()` here? Or have I misunderstood something? :thinking:

Reply:
Yeah I'm stumped by this, I agree that as the comparison is always false there'll be no memoization but I assume Jamie had a reason for doing it. I'm going to remove the memo call for all instances where the comparator is `() => false` and they can be added back in when and where memoization is wanted in the future

Reply:
Thanks for removing all those `React.memo()` wrappers. I'll be interested to hear from Jamie about why he added them. Maybe it has a helpful side-effect apart from memoisation? :thinking:

Reply:
Nope, this wasn't intentional - a side effect of copy & paste oops!